portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,21 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Jacinda's Portfolio</h1>
+      <p>Welcome to my portfolio.</p>
+      <img src = "/images/tenor.gif" />
+      <p>Hi everyone! My full name is Jacinda Ryzel Soto. Right now you are visiting
+        my portfolio page which is kinda cool. I like to code (if that wasn't obvious), read,
+        listen to music, paint, and more!</p>
+      <p>Here is a funny picture of me.</p>
+      <img src = "/images/patrick.gif" />
+      <p>Just kidding! Here is another picture of Patrick.</p>
+      <img src = "/images/200.gif" />
+      <p>Click <a href="https://www.linkedin.com/in/jacindasoto/">here</a> to visit my LinkedIn</p>
+      <p>Click <a href = "/images/LinkedIn pfp.jpg" > here </a> for a picture of me 😄</p>
+      <p>Click below to get random fun facts about me.</p>
+      <button onclick="addRandomGreeting()"> ☺️</button>
+      
```

Comment:
You can probably remove this line, since it doesn't seem to be used for anything. Blank lines are usually used to separate different parts of the source file, but I don't see any particular reason why the greeting-container div needs to be on its own. (Correct me if you feel otherwise!)